glide.yaml

```diff
@@ -35,9 +35,7 @@ import:
 - package: github.com/kardianos/osext
   version: c2c54e542fb797ad986b31721e1baedf214ca413
 - package: go.uber.org/thriftrw
-  version: bbc73fe4e851cac67043d021242eb03920c2f1dd
-  repo:    https://github.com/raynos/thriftrw-go.git
```

Comment:
We were using this thriftrw fork since it added `json:required` tags to struct fields, as they are used by easyjson for validation.

Reply:
v1.6.0 has that feature merged